pkg/controller/daemon/daemoncontroller.go

```diff
@@ -322,7 +322,6 @@ func (dsc *DaemonSetsController) addHistory(obj interface{}) {
 			return
```

Comment:
The comment of this handler is not 100% true, this is also called on controller manager restarts.